.travis.yml

```diff
@@ -56,6 +56,14 @@ jobs:
         - TEST: Shell Check
       script: shellcheck *.sh
 
+    - stage: Test
+      env:
+        - TEST: .travis.yml and .travis.yml consistency
```

Comment:
Should this say ".travis.yml and .travis.yml.template consistency" ?

Reply:
Yes, my fault, thanks!